Sectors/Lines - LTC.txt

```diff
@@ -328,16 +341,58 @@ SECTORLINE:LTC32
 COORD:N050.46.26.000:W000.49.42.000
 COORD:N051.08.53.000:W000.11.25.000
 
-SECTORLINE:LTC33
-DISPLAY:London TC East:London TC East:London AC Clacton
-DISPLAY:London AC Clacton:London AC Clacton:London TC East
-DISPLAY:London AC Dover:London AC Clacton:London TC East
+SECTORLINE:LTC33a
+DISPLAY:London TC DAGGA:London TC DAGGA:London AC Clacton
+DISPLAY:London TC DAGGA:London TC DAGGA:London TC REDFA
+DISPLAY:London AC Clacton:London AC Clacton:London TC DAGGA
+DISPLAY:London AC Clacton:London AC Clacton:London TC REDFA
+DISPLAY:London AC Dover:London AC Clacton:London TC DAGGA
+DISPLAY:London AC Dover:London AC Clacton:London TC REDFA
 COORD:N052.07.13.000:E000.51.49.000
 COORD:N051.48.27.000:E001.36.43.000
+
+SECTORLINE:LTC33b
+DISPLAY:London TC JACKO:London TC JACKO:London AC Clacton
+DISPLAY:London AC Clacton:London AC Clacton:London TC JACKO
+DISPLAY:London AC Dover:London AC Clacton:London TC JACKO
+DISPLAY:London AC Dover:London AC Clacton:London TC SABER
+DISPLAY:London TC JACKO:London TC JACKO:London AC SABER
+DISPLAY:London TC SABER:London TC SABER:London AC JACKO
+DISPLAY:London TC SABER:London TC SABER:London AC Clacton
+DISPLAY:London AC Clacton:London AC Clacton:London TC SABER
+COORD:N051.48.27.000:E001.36.43.000
 COORD:N051.24.30.000:E001.36.43.000
 
-SECTORLINE:LTC34
+SECTORLINE:LTC34a
 COORD:N052.11.04.000:E000.05.28.000
 COORD:N052.11.04.000:E000.32.42.000
 COORD:N052.12.21.000:E000.48.19.000
+
+SECTORLINE:LTC34b
+COORD:N052.12.21.000:E000.48.19.000
 COORD:N052.07.13.000:E000.51.49.000
+
+SECTORLINE:LTC35
+COORD:N051.57.02.160:E002.21.22.620
+COORD:N051.56.19.000:E001.51.02.000
+COORD:N051.48.27.000:E001.36.43.000
+DISPLAY:London TC JACKO:London TC JACKO:London TC REDFA
+DISPLAY:London TC REDFA:London TC REDFA:London TC JACKO
+
+SECTORLINE:LTC36
+COORD:N051.45.03.000:E000.50.32.000
+COORD:N051.48.27.000:E001.36.43.000
+DISPLAY:London TC DAGGA:London TC DAGGA:London TC SABER
```

Comment:
Not that it matters at all, but DISPLAY here is below COORD (just a break with the trend)!